src/odemis/driver/smaract.py

```diff
@@ -3084,3 +3085,1139 @@ def SA_CTL_Move(self, handle, ch, pos_pm, _):
 
     def SA_CTL_Stop(self, handle, ch, _):
         self.stopping.set()
+
+
+class SA_SIDLL(CDLL):
+    """
+    Subclass of CDLL specific to SA_SI library, which handles error codes for
+    all the functions automatically.
+    """
+    SA_SI_TIMEOUT_INFINITE = 0xffffffff
+    SA_SI_STRING_MAX_LENGTH = 63
```

Comment:
On the other hand, the functions SA_SI_Cancel, SA_SI_EPK,... all have this prefix and the prefix also appears for these attributes in the documentation. There is also some events that are a bit different and use the "SA_PS" prefix. I think it might be clearer to keep all prefixes. What do you think?